Docs/docs/avatars/avatar-performance-ranking-system.md

```diff
@@ -260,16 +260,16 @@ When you choose a Performance Rank in VRChat's menu, all avatars that are below
 ### Minimum Displayed Performance Rank on PC
 On VRChat for PC, the Minimum Displayed Performance Rank is set to "Very Poor" by default. This means that, by default, no avatars will have their components or display affected for performance reasons on PC. If you wish to change this, you can choose between "Medium", "Poor", or "Very Poor" options.
 
-### Avatar Performance Rank Blocking on Android
-On VRChat for Android, the Avatar Performance Rank Block is set to "Medium" by default. You can choose to change this to "Poor" to see avatars of that rank, but your performance may suffer as a result.
+### Avatar Performance Rank Blocking on Mobile
```

Comment:
hyperlink to section is broken. I think we should add some id tag here for compatibility.

https://creators.vrchat.com/avatars/avatar-performance-ranking-system/#avatar-performance-rank-blocking-on-android